src/acquisition/covid_hosp/state_daily/update.py

```diff
@@ -21,7 +23,51 @@ def run(network=Network):
       Whether a new dataset was acquired.
     """
 
-    return Utils.update_dataset(Database, network)
+    #return Utils.update_dataset(Database, network)
```

Comment:
this just commented out for context with the below documentation?

Reply:
Yep